docs/reference/datastructures/3-headers.md

```diff
@@ -1,5 +1,38 @@
 # Headers
 
+::: starlite.datastructures.Headers
+    options:
+        members:
+            - __init__
+            - from_scope
+            - from_message
+            - raw
+            - keys
+            - values
+            - items
+            - getlist
+            - mutablecopy
+
+
+::: starlite.datastructures.MutableHeaders
+    options:
+        members:
+            - __init__
+            - from_scope
+            - from_message
+            - raw
+            - keys
+            - values
+            - items
+            - getall
+            - getlist
+            - mutablecopy
+            - setdefault
+            - update
+            - append
+            - add_vary_header
```

Comment:
i usually sort keys alphabetically here, not by the order of the code. 

tip: https://plugins.jetbrains.com/plugin/2162-string-manipulation